modules/monitoring/application/views/scripts/list/servicegrid-flipped.phtml

```diff
@@ -9,6 +9,25 @@ if (! $this->compact): ?>
     <div class="sort-controls-container">
         <?= $this->sortBox ?>
     </div>
+    <?php
+    $toggleFilter = Url::fromRequest();
+    if ($toggleFilter->getParam('problems')) {
+        $title = 'Show entire environment';
+        $toggleFilter->remove('problems');
+    } else {
+        $toggleFilter->setParam('problems');
+        $title = 'Show problem context';
+    }
+    ?>
+    <?= $this->qlink(
+        $this->translate($title),
+        $toggleFilter,
+        null,
+        array(
+            'data-base-target'    => '_self',
+            'title'               => $this->translate('Toggle relevance filter')
```

Comment:
The `data-base-target` is not required here. `_self` is the default behavior and not adjusted otherwise by a parent. Also the `title` is, since the link got a nice label now, rather obsolete. Thus the `qlink` call can be shortened I guess.